src/ui/libs/DatalensChartkit/Error/Error.tsx

```diff
@@ -237,6 +237,12 @@ const ChartKitError: React.FC<any> = (props) => {
                 ) {
                     errorDetails.push(i18n('component.chartkit-error.codes', errorCode));
                     detailsString = source.debug.db_message;
+                }
+                // TODO: Replace with specific code
```

Comment:
Why we didn't make the specific code in this PR?

Reply:
waiting for backend